packages/esm-patient-allergies-app/translations/km.json

```diff
@@ -8,32 +8,32 @@
   "allergyNowVisible": "ឥឡូវនេះវាអាចមើលឃើញនៅលើទំព័រអាឡែស៊ី",
   "allergySaved": "អាឡែរហ្សីត្រូវបានរក្សាទុក",
   "allergySaveError": "កំហុសក្នុងការរក្សាទុកអាឡែរហ្សី",
-  "comments": "មតិយោបល់",
-  "dateAndComments": "កាលបរិច្ឆេទនិងមតិយោបល់",
-  "dateOfFirstOnset": "កាលបរិច្ឆេទនៃការចាប់ផ្តើមដំបូង",
+  "dateOfOnsetAndComments": "Date of onset and comments",
   "discard": "បោះបង់",
   "drug": "ថ្នាំ",
+  "emptyStateText": "There are no {{displayText}} to display for this patient",
   "environmental": "បរិស្ថាន",
   "errorFetchingData": "កំហុសក្នុងការទាញយកអាឡែហ្ស៊ី និងប្រតិកម្ម",
   "food": "អាហារ",
   "invalidComment": "មតិយោបល់មិនត្រឹមត្រូវ សូមព្យាយាមម្តងទៀត",
-  "lastUpdated": "ធ្វើ​បច្ចុប្បន្នភាព​ចុងក្រោយ",
   "loading": "កំពុងផ្ទុក",
   "mild": "ស្រាល",
   "moderate": "មធ្យម",
   "name": "ឈ្មោះ",
+  "onsetDateAndComments": "Onset date and comments",
   "otherNonCodedAllergen": "សារធាតុអាឡែហ្សីនដែលមិនមានកូដផ្សេងទៀត។",
   "otherNonCodedAllergicReaction": "ប្រតិកម្មអាលែហ្សីដែលមិនមានលេខកូដផ្សេងទៀត។",
+  "reaction": "Reaction",
   "reactions": "ប្រតិកម្ម",
+  "record": "Record",
   "saveAndClose": "រក្សាទុក និងបិទ",
-  "seeAll": "មើលឃើញ​ទាំងអស់",
+  "seeAll": "មើលឃើញ\u200bទាំងអស់",
```

Comment:
@vasharma05 Looks like some Unicode characters are getting interpolated into existing translations. Can you look into it?

Reply:
Sure!